aries-site/src/examples/templates/page-layouts/BreakPointsTable.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import {
+  Box,
+  Table,
+  TableCell,
+  TableHeader,
+  TableBody,
+  TableRow,
+  Text,
+} from 'grommet';
+
+const data = [
+  {
+    id: 1,
+    size: (
+      <Text color="text-strong" weight="bold">
+        XSmall
```

Comment:
soft opinion, but we tend to implement tshirt sizes as lowercase -- wondering if we should list them as such as well?
```suggestion
       xsmall
```